### PR TITLE
Added TAILS pool

### PR DESCRIPTION
**Pull request by frog357**

### CODE REVIEW

File: adapools.json [hunk 1]
```diff
@@ -1072,7 +1072,12 @@
         "pool_id": "297b5acd8b90df8a1bfd98aa1a61f323b87fead4b8a2b55f2fb83ce6",
         "member_since": "2021-06-15",
         "name": "ALoop"
+      },
+      "212": {
+        "pool_id": "2ce9bfc4e0a732d21ec8739cbfc3b10957b4e3297fd320d556f48ada",
```
Comment (Bez625) [inline adapools.json:1077]: Please do not manually change `adapools.json` - this is automatically generated

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
@frog357 Please can you make the changes requested? E.g. remove the change to adapools.json Thanks

**Review by frog357 — COMMENTED** [review]
Removed changes to adapools.json

File: adapools.json (hunk 1 above)
Comment (frog357) [reply adapools.json]: > @frog357 Please can you make the changes requested? E.g. remove the change to adapools.json Thanks

You originally said "Please do not manually change adapools.json - this is automatically generated" This was not a statement telling me to take action.

Comment (Bez625) [reply adapools.json]: I added this comment and marked the review as "Request changes". Apologies if you are new to github, but this is generally a reviewers way of indicating that changes need to be made. I think the combined status of the PR and the comment I left made it pretty clear what needed to happen. Either way, you got there in the end!

**Pull request merged**